Sounds/wwise_project/wwise_project.mrole.validationcache

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ValidationCache Version="1" SchemaVersion="103">
```

Comment:
These `*.wsettings` files can be ignored & removed.

Reply:
Done, thanks!